irohad/ametsuchi/impl/storage_impl.cpp

```diff
@@ -316,7 +320,7 @@ namespace iroha {
                   },
                   [](auto &&e) -> expected::Result<
                                    std::shared_ptr<const iroha::LedgerState>,
-                                   std::string> { return e; });
+                                   std::string> { return std::move(e); });
```

Comment:
```
irohad/ametsuchi/impl/storage_impl.cpp:326:58: warning: local variable 'e' will be copied despite being returned by name [-Wreturn-std-move]
                                   std::string> { return e; });
<snip>
irohad/ametsuchi/impl/storage_impl.cpp:326:58: note: call 'std::move' explicitly to avoid copying
                                   std::string> { return e; });
                                                         ^
                                                         std::move(e)
```
```
clang++ --version
clang version 7.0.1-8 (tags/RELEASE_701/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```